utils/xmllite.d

```diff
@@ -395,6 +411,9 @@ void parseInto(Config)(XmlDocument d, ref StringStream s)
 void parseInto(Config)(XmlNode node, ref StringStream s)
 {
 	node.startPos = s.position;
+
+	auto leading_whitespace_backup = s.backup;
```

Comment:
`leading_whitespace_backup` is used only in a `static if`-block. It can be defined in a `static if`-block as well:

```d
static if (Config.keepWhitespace)
{
    auto leading_whitespace_backup = s.backup;
}
```